parcels/field.py

```diff
@@ -351,7 +362,8 @@ def from_netcdf(cls, filenames, variable, dimensions, indices=None, grid=None,
                     else:
                         data_list.append(buffer_data)
                 ti += len(tslice)
-            data = da.concatenate(data_list, axis=0)
+            lib = np if isinstance(data_list[0], np.ndarray) else da
+            data = lib.concatenate(data_list, axis=0)
```

Comment:
This was a bug before, that it was always using `da`?

Reply:
Indeed I've just found it yesterday. It was not a problem, but basically it was converting np data to dask where it was not necessary (maybe bringing some memory overhead)